src/app/shared/plnkr/plunker-writer.ts

```diff
@@ -0,0 +1,128 @@
+import { Http } from '@angular/http';
+import { PlunkerData } from './plunker-data';
+
+/**
+ * Plunker writer, write example files to Plunker
+ *
+ * Plunker API
+ * URL: http://plnkr.co/edit/?p=preview
+ * data: {
+ *   // File name, directory and content of files
+ *   files[file-name1]: file-content1,
+ *   files[directory-name/file-name2]: file-content2,
+ *   // Can add multiple tags
+ *   tags[0]: tag-0,
+ *   // Description of plunker
+ *   description: description,
+ *   // Private or not
+ *   private: true
+ * }
+ */
+export class PlunkerWriter {
+  PLUNKER_URL: string = 'http://plnkr.co/edit/?p=preview';
+
+  COPYRIGHT =
+    `Copyright 2016 Google Inc. All Rights Reserved.
+    Use of this source code is governed by an MIT-style license that
+    can be found in the LICENSE file at http://angular.io/license`;
+
+  TEMPLATE_PATH = '/app/shared/plnkr/template/';
+  TEMPLATE_FILES = ['index.html', 'systemjs.config.js', 'main.ts'];
+
+  TAGS: string[] = ['angular', 'material', 'example'];
```

Comment:
I think you missed moving these constants out of the `class`; they can go right above it ([example](https://github.com/angular/material2/blob/8caf9a6553d57f46ea88a4e306a5438bb2dac571/src/lib/slider/slider.ts#L31-L31))